server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -211,6 +211,18 @@ data ScheduledEventType =
   -- so all the configuration is fetched along the scheduled events.
     deriving (Eq, Show)
 
+data ScheduledEventWebhookBody
+  = ScheduledEventWebhookBody
```

Comment:
I’m a little bit curious: the original issue shows a request payload with these fields in the console. So these fields must come from somewhere in the code already, right? It seems like it would be useful to share the definition of those types in some way. But maybe I’m misunderstanding something.

Reply:
I think it is coming from here: https://github.com/hasura/graphql-engine/blob/9b247d4ed79e6e82409d994c70b52c561c832230/server/src-lib/Hasura/Eventing/HTTP.hs#L69

Reply:
I think we could potentially change `_rqPayload` -> `_rqBody` to stay consistent.

Reply:
The request that's being shown in the console are from the invocation logs. While logging the request and response, the earlier behaviour was to log everything related to the scheduled event, which included un-necessary data like the `retry_conf`,`header_conf`. 

In this PR, I have made the change in the request body and while logging the invocation, only the request that has been sent will be included, nothing more.

Reply:
Currently, the `WebhookRequest` is being used by Event triggers also, that's why I was hesitant to make the `_rqPayload`-> `_rqBody` change. I'll add a new type for scheduled triggers to use `body` instead of `payload`.

Reply:
I wonder if changing the invocation log format for event triggers would be an invasive change. I would like to hear more opinions but I am in favour of changing the name to `body` for both. It is clearer for the user as well.

Reply:
I don’t really have any opinion about whether we go with `body` or `payload`, but I do think we should be consistent about it! And if we already use `body` in event triggers, it sounds like that sort of forces our hand due to backwards-compatibility constraints. So I would be in favor of sharing the datatype and request format if at all possible.

Reply:
…wait, no, I think I misunderstood. It sounds like event triggers already use `payload`, right? Not `body`. In which case everything I said still applies, except that we’re forced to use `payload` instead of `body`.

Reply:
Okay, from a conversation on Slack, we worked out the confusion: I was trying to talk about the event trigger request payload, but the event trigger request payload is meaningfully different, so there’s no sharing opportunities there. Given that, I’m fine with this PR.